modules/solid_mechanics/test/tests/ad_anisotropic_creep/ad_aniso_creep_xy_3d_anisoElasticity.i

```diff
@@ -0,0 +1,283 @@
+# This test simulates biaxial tensile test with the material being anisotropic
+# in terms of elasticity and creep. This test demonstrates that the implement-
+# ed algorithm, that allows anisotropic elasticity, converges to a solution.
```

Comment:
Is this the best we have?  In other words, is there an analytic solution for a case like this?

Reply:
Added the analytical solution to both the tests.